src/frontend/demo/src/App.tsx

```diff
@@ -33,6 +34,8 @@ export default function App() {
         <Route path="/groups" element={<GroupsView />} />
         <Route path="/rooms" element={<RoomsView />} />
         <Route path="/rooms/:slug" element={<RoomView />} />
+        <Route path="/j/m/:meetingId" element={<JitsiView />} />
+        <Route path="/j/:roomSlug" element={<JitsiView />} />
```

Comment:
Why such short path ?

Reply:
In my opinion, to keep the url as simple as possible (meeting.education/j/my-amazing-room) but still avoiding potential conflicts, so we don't need to forbid some room names because it's the name of other rooms

Reply:
Should we close that or would you prefer me to update the path. What are your suggestions ? Thank you